universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,31 @@
+# --- JSON-based tables
+
+# --- !Ups
+create table if not exists applicants (
+  id bigserial primary key,
+  object jsonb not null
+  -- No other constraint on applicant - empty object acceptable.
+);
+
+create table if not exists questions (
+  id bigserial primary key,
+  object jsonb not null,
+  check (object ? 'target')
```

Comment:
question: What does this expression mean? That the object must contain a field called 'target'?

Reply:
Yep!  Added comment.